src/features/authentication/components/LoginForm.tsx

```diff
@@ -7,16 +7,22 @@ import logo from "../../../assets/Logo.png";
 import { AuthServices } from "../services";
 import type { ILogin } from "../models/Login";
 import { AxiosError } from "axios";
+import { useNavigate } from "react-router-dom";
 
 const LoginForm = () => {
+    const navigate = useNavigate();
     const [data, setData] = useState<ILogin>({ email: "", password: "" });
-
     const onLogin = async () => {
         const response = await AuthServices.logIn(data);
-        if (response instanceof AxiosError) return;
-        // TODO: Handle redirect to home page with router here
+        if (response instanceof AxiosError)
+            // -- error in login
+            // console.log(response.response?.data);
+            return;
```

Comment:
if the only only returns a null value, probably its not necessary, or should be handled in other way..